shlex.js

```diff
@@ -260,7 +260,7 @@ class Shlexer {
 
 
 /**
- * Splits a given string using shell-like syntax.
+ * Splits a given string using shell-like syntax. This function is the inverse of shlex.join().
```

Comment:
Can you please wrap all of the lines at 80 columns to match the rest of the code?